### ISSUE TITLE
Parameterized tests not evaluated correctly in grader

### ISSUE THREAD

**Issue by Danghor** [sourcegrade/jagr#140]
When using parameterized tests, the grader will give points when the test passes for the first parameter set. It is expected that the tests fails overall if any of the parameter sets makes the test fail. How to reproduce:

Test:
```java
@ParameterizedTest
@CsvFileSource(resources = "/Test.csv", numLinesToSkip = 1)
public void test(boolean value) {
    assertEquals(true, value);
}
```

With CSV:
```csv
foo
true
false
```

The test will fail in IntelliJ
<img width="310" alt="image" src="https://user-images.githubusercontent.com/7728268/196983179-fbc9aab9-852f-47f3-aa03-48bfab588b10.png">

but it will pass in the grader:
<img width="1124" alt="image" src="https://user-images.githubusercontent.com/7728268/196983325-6cb60169-b65d-4694-b77a-280877e35b35.png">

The issue can be reproduced on this branch: https://github.com/FOP-2223/FOP-2223-H03-Root/tree/parameterizedBug

**Comment by Danghor** [comment]
The same thing happens when using `@ValueSource(booleans = {true, false})`. It will fail in IntelliJ, but pass in the grader.
When using `@ValueSource(booleans = {false, true})` instead, it will fail both in IntelliJ and in the grader.

**Comment by Danghor** [comment]
Using `id("org.sourcegrade.jagr-gradle") version "0.6.1"`

**Comment by dst97** [comment]
Also affected `JsonClasspathSource`. Problem is fixed with 0.6.2.
